mutt segfaults everytime the 'unmailboxes' command is used 
(see upstream #3273)

--- a/buffy.c
+++ b/buffy.c
@@ -236,7 +236,6 @@
     {
       if(*tmp)
       {
-        FREE (&((*tmp)->path));
         tmp1=(*tmp)->next;
         FREE (tmp);		/* __FREE_CHECKED__ */
         *tmp=tmp1;
